make.cmd

```diff
@@ -126,14 +126,14 @@ msbuild /t:DistClean /p:BuildFlavour=Debug /verbosity:minimal /nologo /p:Platfor
 goto :main
 
 :ngen
-%SystemRoot%\Microsoft.NET\Framework\v4.0.30319\ngen.exe install bin\Release\ipy.exe
+%SystemRoot%\Microsoft.NET\Framework\v4.0.30319\ngen.exe install bin\Release\net45\ipy.exe
 goto :exit
 
 :main
 msbuild Build.proj /t:%_target% /p:BuildFlavour=%_flavour% /verbosity:minimal /nologo /p:Platform="Any CPU"
 if "%_target%" == "Build" (
     echo "Copying %_flavour% test dlls"
-    xcopy /y /q Src\DLR\bin\%_flavour%\*.dll bin\%_flavour%\
+    xcopy /y /q Src\DLR\bin\%_flavour%\net45\*.dll bin\%_flavour%\
```

Comment:
The destination probably needs to be `bin\%_flavour%\net45`. Not a great solution for multi-targetting but unless someone has a better idea to deploy the DLLs I suggest we fix it in a separate PR.